articles/install-guide/update.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Learn how to update the Microsoft Quantum Development Kit (QDK)
+author: natke
+ms.author: nakersha
+ms.date: 9/30/2019
+ms.topic: article
+ms.custom: how-to
+uid: microsoft.quantum.update
+---
+
+# Update the Microsoft Quantum Development Kit (QDK)
+
+Learn how to update the Microsoft Quantum Development Kit (QDK) to the latest version.
+
+This article assumes that you already have the QDK installed. If you are installing for the first time, then please refer to the [installation guide](xref:microsoft.quantum.install)
+
+The update steps depend on your development environment. Choose your environment from the following sections.
+
+## Python
+
+1. Update the `iqsharp` kernel
+
+    ```bash
+    dotnet tool update -g Microsoft.Quantum.IQSharp
+    dotnet iqsharp install
+    ```
+
+1. Verify the `iqsharp` version
+
+    ```bash
+    dotnet iqsharp --version
+    ```
+
+    You should see the following output:
+
+    ```bash
+    iqsharp: 0.9.1909.3002
+    Jupyter Core: 1.1.14623.0
+    ```
+
+1. Update the `qsharp` package
+
+    ```bash
+    pip install qsharp --upgrade
+    ```
+
+1. Verify the `qsharp` version
+
+    ```bash
+    pip show qsharp
+    ```
+
+    You see the following output:
+
+    ```bash
+    Name: qsharp
+    Version: 0.9.1909.3002
+    Summary: Python client for Q#, a domain-specific quantum programming language
+    ...
+    ```
+
+1. You can now use the updated QDK version to run your existing quantum programs.
+
+## Jupyter notebooks
+
+1. Update the `iqsharp` kernel
+
+    ```bash
+    dotnet tool update -g Microsoft.Quantum.IQSharp
+    dotnet iqsharp install
+    ```
+
+1. Verify the `iqsharp` version
+
+    ```bash
+    dotnet iqsharp --version
+    ```
+
+    You should see the following output:
+
+    ```bash
+    iqsharp: 0.9.1909.3002
+    Jupyter Core: 1.1.14623.0
+    ```
+
+1. You can now open an existing Jupyter notebook and run it with the updated QDK.
+
+## C# on Windows, using Visual Studio
+
+1. Update the Q# Visual Studio extension
+
+    - Visual Studio prompts you to accept updates to the [Quantum Visual Studio extension](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit)
+    - Accept the update
+
+    > [!NOTE]
+    > The project templates are updated with the extension. The updated templates apply to newly created projects only. The code for your existing projects is not updated when the extension is updated.
+
+1. Update the QDK packages
+
+    - Open an existing application
+    - Select **Dependencies** in the Solution Explorer
+    - Select **Manage NuGet Packages**
+    - Update the **Microsoft.Quantum** packages to the latest version
+
+1. You can now run your application with the latest QDK
+
+## C#, using VS Code
+
+1. Update the Quantum VS Code extension
+
+    - Restart VS Code
+    - Navigate to the **Extensions** tab
+    - Select the **Microsoft Quantum Development Kit for Visual Studio Code** extension
```

Comment:
These steps should happen automatically unless the user has opted out from VS Code's autoupdate functionality.

Reply:
I tested this on my own installation, and it didn't happen automatically.  (And I don't think I have opted out of auto updates. Let me check though

Reply:
That's very odd, I wonder if the extension hadn't gone live yet?